cpp/demo/interpolation_different_meshes/main.cpp

```diff
@@ -66,6 +66,13 @@ int main(int argc, char* argv[])
     // Interpolate from u_tet to u_hex
     u_hex->interpolate(*u_tet);
 
+    // Cache data explicitly and use it to interpolate quickly
+    u_hex->x()->set(0);
+    auto m2mInterpolationData
```

Comment:
I should have been more specific; we don't use camel case for variable names e.g. `m2m_interpolation_data`.

Reply:
We still have 'm2m' and 'nmm'. Could we standardise on 'non_matching_meshes' and 'nmm'?